amplify.yml

```diff
@@ -4,17 +4,17 @@ applications:
       phases:
         preBuild:
           commands:
-            - nvm install 14
-            - nvm use 14
+            - nvm install 16.13
```

Comment:
In the past this has broken the Amplify Hosting build. It should be fine since you're deploygin this branch, but let's just keep an eye on this when merged to main

Reply:
16.13+ does break Amplify Hosting. 

Yea, I'll keep an eye on it 👍

Reply:
done